media-sound/pasystray/pasystray-0.7.0.ebuild

```diff
@@ -0,0 +1,53 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit gnome2-utils autotools vcs-snapshot xdg
+
+DESCRIPTION="PulseAudio system tray"
+HOMEPAGE="https://github.com/christophgysin/pasystray"
+SRC_URI="https://github.com/christophgysin/${PN}/archive/${P}.tar.gz"
+
+LICENSE="LGPL-2.1+"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~x86"
+IUSE="libnotify zeroconf"
+
+RDEPEND="
+	>=dev-libs/glib-2.48.2
+	>=media-sound/pulseaudio-5.0-r3[glib,zeroconf?]
+	zeroconf? ( >=net-dns/avahi-0.6 )
+	x11-libs/gtk+:3
+	x11-libs/libX11
+	libnotify? ( >=x11-libs/libnotify-0.7 )
+"
+DEPEND="${RDEPEND}
+	virtual/pkgconfig"
+
+src_prepare() {
+	default
+	eautoreconf
+}
+
+src_configure() {
+	econf \
+		$(use_enable libnotify notify) \
+		$(use_enable zeroconf avahi)
+}
+
+pkg_preinst() {
+	xdg_pkg_preinst
+	gnome2_icon_savelist
```

Comment:
calling this is unnecessary, and dropping it enables you to drop the whole `pkg_preinst` bloc.